airbyte-integrations/connectors/source-salesforce/unit_tests/conftest.py

```diff
@@ -99,6 +99,12 @@ def stream_api_v2(stream_config):
     return _stream_api(stream_config, describe_response_data=describe_response_data)
 
 
+@pytest.fixture(scope="module")
```

Comment:
I suggest this fixture to be decalred in the `api_test.py` module since it's only used there. And I'm not sure the module scope is require

Reply:
I just wanted to be consistent with fixture `stream_api_v2` which is also used only in `api_test.py`